.github/workflows/Tests.yaml

```diff
@@ -858,17 +858,15 @@ ${{ matrix.build_type }}-pch-${{ matrix.use_pch || 'ON' }}"
   # Build all test executables and run unit tests on macOS
   unit_tests_macos:
```

Comment:
Can you please make 2 macOS tests: i) Apple Silicon, and ii) Intel?

Reply:
Oh I don't think we need Intel anymore, do we?

Reply:
People still use Intel Macs. They aren't that old yet so I think we should continue to test them